src/Hipchat.php

```diff
@@ -43,7 +43,10 @@ function postOutput($outputString)
             echo $outputString;
         } else {
             $outputString = $this->convertNewLineToBr($outputString);
-            $this->postInHipChat($outputString);
+            if (!$this->wasLastMessageTheSameAsThisMessage($outputString))
```

Comment:
hrmmm.  Maybe we dont run it as often on weekends?  I think during the week, it's important to get the nags until someone fixes it.